packages/Uploader/CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Add
+
+- OnError prop, now give the ability to the consumer to manipulate the error message per file in case of need it.
+
+### Remove
+
+- The ability to retry and upload a file has been remove, was not stable and need more time to make it work properly
+
+### Changed
+
+- `breaking change` Now whenever you use the **onRequest** prop, you should switch from `onError` function to `onEnd` function which is what really is happening
+
+```js
+        <Uploader
+          {...}
+          onRequest={({ file, onProgress, onEnd, formData }) => {
+            file.request
+              .get("https://api.example.com:4001/")
+              .withCredentials()
+              .on("progress", onProgress)
+              .end(onEnd);
+          }}
```

Comment:
This would be helpful to have in the documentation too, at least in the README.

Reply:
Oh, I see now it's in one of the stories.  That should be sufficient actually, you can disregard the above recommendation.